bee-ledger/src/workers/pruning/error.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message::{prelude::MilestoneIndex, MessageId};
+
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Minimum: {minimum}, Found: {found}")]
+    InvalidTargetIndex {
+        minimum: MilestoneIndex,
+        found: MilestoneIndex,
+    },
+
```

Comment:
do we need these spaces?

Reply:
you mean the empty lines between variants?